.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,ts,html}]
+charset = utf-8
+indent_style = tab
+indent_size = 2
```

Comment:
If this is merged I don't think the config should enforce indent width considering it is only a visual configuration and doesn't affect the file written to disk. I often use 4 columns to indent, for example.

Reply:
Sure, don't hesitate to tweak it to your needs, I just wanted to start the debate..
You guys know this codebase much more than I do ! :bowing_man: